src/Game.cxx

```diff
@@ -263,6 +269,16 @@ void Game::run(bool SkipMenu)
     Engine::instance().newGame();
   }
 
+  GameClock &gameClock = GameClock::instance();
+
+  gameClock.addRealTimeClockTask(
+  [this]()
+  {
+    Game::updateZones();
+    return false;
```

Comment:
The GameClock can be set to trigger action periodically. In that case it will be always active until it is canceled. There is 2 ways to do that. First is to store timer handle which is returned by addRealTimeClockTask function, and call removeClockTask providing the handle. Second option is using the return value of the lambda action. If the action returns true, the timer will be canceled and not repeted anymore.